modin/spreadsheet/test/test_general.py

```diff
@@ -0,0 +1,58 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import pandas
+import pytest
+import modin.pandas as pd
+import modin.spreadsheet as mss
+import numpy as np
+from modin_spreadsheet import SpreadsheetWidget
+
+
+def get_test_data():
+    return {
+        "A": 1.0,
+        "B": pd.Timestamp("20130102"),
+        "C": pd.Series(1, index=list(range(4)), dtype="float32"),
+        "D": np.array([3] * 4, dtype="int32"),
+        "E": pd.Categorical(["test", "train", "foo", "bar"]),
+        "F": ["foo", "bar", "buzz", "fox"],
+    }
+
+
+def test_from_dataframe():
+    data = get_test_data()
```

Comment:
@devin-petersohn I get NotImplementedError on this line for the CI tests, should I not be using helper methods?

Reply:
This is the error: https://github.com/modin-project/modin/pull/2797/checks?check_run_id=2024302005

Reply:
I think you just need to remove experimental flag. See my review above.

Reply:
Looks like it's still failing: https://github.com/modin-project/modin/pull/2797/checks?check_run_id=2024670302